### PR TITLE
Release updates

### PR DESCRIPTION
**Pull request by jessfraz**
2 things:
-  add script to generate index listing for the repo if say hosted on s3
-  add clean apt db to release script

### CODE REVIEW

File: hack/make/generate-index-listing [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		# output with correct format
+		sed -e 's,\ @.*$, '"$F"',g'<<<"$L";
+	done;
+	) >> $index_file;
+
+	# now output a list of all directories in this dir (maxdepth 1) other than '.' outputting in a sorted manner exactly like apache
+	find -L . -mount -depth -maxdepth 1 -type d ! -name '.' -printf "<a href=\"%f\">%-43f@_@%Td-%Tb-%TY %Tk:%TM  -\n"|sort -d|sed 's,\([\ ]\+\)@_@,/</a>\1,g' >> $index_file
+
+	# print the footer html
+	echo "</pre><hr></body></html></body></html>" >> $index_file
```
Comment (thaJeztah) [inline hack/make/generate-index-listing]: there's one too many '</body></html>` here

File: hack/make/generate-index-listing [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	# the index file to create
+	local index_file="${directory}/index"
+
+	# cd into dir & touch the index file
+	cd $directory
+	touch $index_file
+
+	# print the html header
+	cat <<-EOF > "$index_file"
+	<html>
```
Comment (thaJeztah) [inline hack/make/generate-index-listing]: jus-a-suggestion® add `<!DOCTYPE html> ` ?

File: hack/make/generate-index-listing (hunk 1 above)
Comment (jfrazelle) [inline hack/make/generate-index-listing]: lol i html all the time can you tell

**Comment by jfrazelle** [comment]
ping @tianon

**Comment by jfrazelle** [comment]
@tianon :innocent: :heart: :heart: :heart: :heart:

File: hack/make/generate-index-listing [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	<h1>Index of ${cleaned}/</h1><hr>
+	<pre><a href="../">../</a>
+	EOF
+
+	# start of content output
+	(
+	# change IFS locally within subshell so the for loop saves line correctly to L var
+	IFS=$'\n';
+
+	# pretty sweet, will mimick the normal apache output
```
Comment (tianon) [inline hack/make/generate-index-listing]: Hahahaha you're awesome :+1:

**Comment by tianon** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Pull request merged**